### ISSUE TITLE
Question about SimCLR

### ISSUE THREAD

**Issue by muzhibenying** [binli123/dsmil-wsi#6]
Should I use the patches in test set when I train the SimCLR model?

**Comment by binli123** [comment]
Ideally no. You could try to exclude the patches extracted from the testing images, but the impact is probably minor since the number of patches from the training set alone is more than enough for learning good representations.
